### ISSUE TITLE
Taking

### ISSUE THREAD

**Issue by thejohnhoffer** [dylanbeattie/rockstar#14]
"taking" doesn't sound so great for function calls.

Did you consider using "with"?

Looking Back takes Me and All
Fear is Me
Until Fear is nothing then
   Knock down Fears
   The Way is Looking Back with Fear and Me
   Until The Way is nothing then
     Knock down the Way
     Build up All
Give back All

That Guy is 22
All Of It is nothing
Looking Back with That Guy, All Of It

I think this is what I mean:
```Python
def LookingBack(Me, All):
  Fear = Me
  while Fear != 0:
    Fear -= 1
    TheWay = LookingBack(Fear, Me)
    while TheWay != 0:
      TheWay -= 1
      All += 1
  return All

ThatGuy = 22
AllOfIt= 0
LookingBack(ThatGuy, AllOfIt)
```

**Comment by thejohnhoffer** [comment]
For the curious, this example  could be cleaned up in python like this:
```
def LookingBack(Pain, All):
  for Fear in range(Pain):
    All += LookingBack(Fear, Pain)
  return All
```
For some reason, this is [this sequence](https://oeis.org/A050488)

**Comment by dylanbeattie** [comment]
Yeah, it's hard coming up with a suitably poetic syntax. I'm wondering if we could just omit the keyword altogether and use VBScript style syntax where you give the function name and a comma-separated list of arguments. I think more thinking required on this one.

**Comment by stretchyboy** [comment]
Another issue suggests wants / wanting as aliases for takes/taking.
